### PR TITLE
redirect from removed cls-forcesched.html page to appropriate section in schedulers docs

### PR DESCRIPTION
**Pull request by rutsky**
Needed for https://github.com/buildbot/buildbot/pull/2145

Related issue http://trac.buildbot.net/ticket/3530

**Haven't tested at all! Will this work as I expect?**

### CODE REVIEW

File: files/www-redirects [hunk 1]
```diff
@@ -17,3 +17,5 @@ rewrite ^/buildbot/docs/(latest|current)/WithProperties.html http://docs.buildbo
 rewrite ^/buildbot/docs/(latest|current)/release-notes.html http://docs.buildbot.net/$1/relnotes/index.html permanent;
 rewrite ^/buildbot/docs/(latest|current)/release-notes/(.*) http://docs.buildbot.net/$1/relnotes/$2 permanent;
 rewrite ^/buildbot/docs(.*)$ http://docs.buildbot.net$1 permanent;
+# Renamed documentation pages.
+rewrite ^/latest/developer/cls-forcesched.html http://docs.buildbot.net/latest/manual/cfg-schedulers.html#forcescheduler-scheduler permanent;
```
Comment (sa2ajj) [inline files/www-redirects]: this somehow looks different than the lines above.  Are you **sure** that `^/buildbot/docs/` prefix is not needed?

Comment (rutsky) [inline files/www-redirects]: Yes, I'm sure. Line above are redirections from urls like `buildbot.net/buildbot/docs/...` to `docs.buildbot.net/...`.

I want to redirect URL already from `docs.buildbot.net`.

**Comment by rutsky** [comment]
Please do not merge this PR before https://github.com/buildbot/buildbot/pull/2145 PR is merged.
And most probably it will not be merged, since @tardyp says that ForceScheduler class page should not be removed, but rather edited.

**Comment by rutsky** [comment]
`cls-forcesched.rst` has been updated, not removed. So no need in redirection.

**Pull request closed without merging**